### ISSUE TITLE
TrivialAccessors incorrectly detects offence

### ISSUE THREAD

**Issue by jurriaan** [rubocop/rubocop#136]
The following code triggers the bug:

``` ruby
class TrivialTest
  def test=(val)
    @test = val
    some_function_call
    or_more_of_them
  end
end
```

I get the following error:

```
== test.rb ==
C:  2: Use attr_writer to define trivial writer methods.

1 file inspected, 1 offence detected
```

Which is incorrect because this is not a trivial writer method.
